homeassistant/components/rapt_ble/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "domain": "rapt_ble",
+  "name": "RAPT Pill BLE",
```

Comment:
We should drop Pill here since we may want to add support for additional devices the vendor makes in the future

Also change BLE to Bluetooth since BLE is less likely to be understood by all speakers

Reply:
I’m not against this, limitations/compatible/tested devices can be explained/listed in the doco

Reply:
Okay, done. I've also updated the docs accordingly.